### ISSUE TITLE
Scan 'imports' and 'fully qualified classes' dependencies as well

### ISSUE THREAD

**Issue by qoomon** [classgraph/classgraph#329]
Scan imports and fully qualified class dependencies as well

**Comment by lukehutch** [comment]
Hi, can you please clarify your request?

**Comment by qoomon** [comment]
ClassInfo do not include following dependencies after scan. `LinkedList` and `Predicate` are not listed
```java
import ava.util.function.Predicate;
import java.util.*
class Foo {
  Foo(){
    new<Predicate>();
  }
}
```

**Comment by lukehutch** [comment]
Please show your scanning code too. But if you want to include system classes in the scan results, you need to call `.enableSystemJarsAndModules()`. Does that solve your issue?

**Comment by qoomon** [comment]
Thank you for your suggestion but I've already enabled System and Modules scanning. It does not work with my own Classes in the same package.
```java
package my.package;

import my.package.Bar;
class Foo {
  Foo(){
    new Bar();
  }
}
```

**Comment by lukehutch** [comment]
@qoomon Please provide a complete example that can be run to reproduce the problem. What you have provided is not sufficient to determine the source of the problem.

**Comment by qoomon** [comment]
```java
package my.app;
import my.app.Bar;
public class Foo {
    public Foo() {
        new Bar();
    }
}
```
```java
package my.app;
public class Bar {
}
```
```java
package my.app;
import io.github.classgraph.ClassGraph;
import io.github.classgraph.ClassInfo;
import io.github.classgraph.ScanResult;
public class App {
    public static void main(String[] args) {
        try (ScanResult scanResult =
                     new ClassGraph()
                             .enableAllInfo()
                             .enableInterClassDependencies()
                             .enableExternalClasses()
                             .whitelistPackages("my.app")
                             .scan()) {
            for (ClassInfo classInfo : scanResult.getAllClasses()) {
                System.out.println(classInfo);
                System.out.println(classInfo.getClassDependencies());
            }
        }
    }
}
```

**Comment by lukehutch** [comment]
@qoomon Ah, thanks for the code example. There was a line that got dropped a few versions ago during refactoring. Fixed and released in version 4.8.13.

**Comment by qoomon** [comment]
thx
